www/mkdocs.yml

```
@@ -91,6 +91,8 @@ nav:
     - customization/snapcraft.md
     - customization/docker.md
     - customization/docker_manifest.md
+  - Software Bill of Materials:
+      - Cataloging artifacts: customization/sbom.md
```

this could probably be just:

```suggestion
  - customization/sbom.md
```